app/services/workflow_service.py

```
@@ -0,0 +1,350 @@
+            if not workflow:
+                raise ValueError(f"Workflow {workflow_id} not found")
+            
+            # Get recent executions
+            recent_executions = db.query(ExecutionLog).filter(
+                ExecutionLog.workflow_id == workflow_id
+            ).order_by(ExecutionLog.created_at.desc()).limit(10).all()
+            
+            # Get active triggers
+            active_triggers = db.query(WorkflowTrigger).filter(
+                WorkflowTrigger.workflow_id == workflow_id,
+                WorkflowTrigger.is_active == True
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `WorkflowTrigger.is_active:` for truth checks

Replace with `WorkflowTrigger.is_active`

(E712)